<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить все значения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_org_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add org</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_employee_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add empl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_phone_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add phone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_journal_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add journal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_employee_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show empl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_org_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show org</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_phone_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show phone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_journal_button">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show journal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_choosen_org_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод номеров организации
(введите название организации)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditShowOrg">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_emp_service">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>350</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Список услуг сотрудника
(введите ФИО)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_org_service">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Список услуг организации
(суммарно)
(введите название услуги)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить тестовые значения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_emp_service_sum">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Список услуг сотрудника
(суммарно)
(введите название услуги)</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>приветик</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
